console.py

```diff
@@ -3,7 +3,7 @@
 
 import cmd
 from datetime import datetime
-import models
+from models.__init__ import storage
```

Comment:
Should just be able to do `from models import storage`.

Reply:
Yes, go this updated!  Thank you!